sql/database.sql

```diff
@@ -9555,19 +9555,29 @@ CREATE TABLE `form_eye_mag_wearing` (
 
 DROP TABLE IF EXISTS `form_taskman`;
 CREATE TABLE `form_taskman` (
-    `ID` bigint(20) NOT NULL AUTO_INCREMENT, 
-    `REQ_DATE` datetime NOT NULL, 
-    `FROM_ID` bigint(20) NOT NULL, 
-    `TO_ID` bigint(20) NOT NULL, 
-    `PATIENT_ID` bigint(20) NOT NULL, `DOC_TYPE` varchar(20) DEFAULT NULL, 
-    `DOC_ID` bigint(20) DEFAULT NULL, 
-    `ENC_ID` bigint(20) DEFAULT NULL, 
-    `METHOD` varchar(20) NOT NULL, `COMPLETED` varchar(1) DEFAULT NULL COMMENT '1 = completed', 
-    `COMPLETED_DATE` datetime DEFAULT NULL, 
-    `COMMENT` varchar(50) DEFAULT NULL, 
-    `USERFIELD_1` varchar(50) DEFAULT NULL, 
+    `ID` bigint(20) NOT NULL AUTO_INCREMENT,
+    `REQ_DATE` datetime NOT NULL,
+    `FROM_ID` bigint(20) NOT NULL,
+    `TO_ID` bigint(20) NOT NULL,
+    `PATIENT_ID` bigint(20) NOT NULL, `DOC_TYPE` varchar(20) DEFAULT NULL,
+    `DOC_ID` bigint(20) DEFAULT NULL,
+    `ENC_ID` bigint(20) DEFAULT NULL,
+    `METHOD` varchar(20) NOT NULL, `COMPLETED` varchar(1) DEFAULT NULL COMMENT '1 = completed',
+    `COMPLETED_DATE` datetime DEFAULT NULL,
+    `COMMENT` varchar(50) DEFAULT NULL,
+    `USERFIELD_1` varchar(50) DEFAULT NULL,
     PRIMARY KEY (`ID`)
 ) ENGINE=INNODB;
 
 -- End of Eye Module tables
 -- ----------------------------------------------------
+--
+-- Table structure for table 'product_registration'
+--
+CREATE TABLE `product_registration` (
+  `registration_id` char(36) NULL DEFAULT '',
```

Comment:
Regarding the error I just posted, looks like just missing the NOT in front of NULL:
(I can do this minor change when bring the code in)